setup.py

```diff
@@ -196,13 +199,11 @@ def check_system_cfitsio_objects(self, obj_name):
                 else:
                     return False
 
-include_dirs=[numpy.get_include()]
-
 sources = ["fitsio/fitsio_pywrap.c"]
 data_files=[]
 
-ext=Extension("fitsio._fitsio_wrap", 
-              sources, include_dirs=include_dirs)
+ext=Extension("fitsio._fitsio_wrap",
+              sources, include_dirs=['numpy'])
```

Comment:
This is just a marker that gets replaced in `build_ext.run` (`extension.include_dirs.index('numpy')`). This could be simplified as you have only one extension, but this is more generic and also a mean to show that the extension requires numpy in its definition.